app/importers/import_task.rb

```diff
@@ -1,38 +1,45 @@
 class ImportTask
-  def initialize(options:)
-    @options = options
+  # If passed invalid arguments, validete and fail in the constructor
+  def initialize(options = {})
+    @options = options.symbolize_keys
 
-    # options["school"] is an optional filter; imports all schools if nil
-    @school = @options.fetch("school", nil)
+    # which schools to include
+    @school_local_ids = validate_school_local_ids!(@options[:school_local_ids])
 
-    # options["source"] describes which external data sources to import from
-    @source = @options.fetch("source", ["x2", "star"])
+    # which importers to run
+    @importer_descriptions = validate_importer_keys!(@options[:importer_keys])
```

Comment:
The substantive change here is to require callers to pass in all `importer_keys` they want to run instead of only the "x2" and "star" groupings.